daniellamarr/warren-web
# Add image that was missing in the About section

#### What does this PR do?
- In the UI that was used to set up the website, an image was missed. The aim of this PR is to refactor the about section to allow the image to exist in it.
#### Description of Task to be completed?
-  Add about image
#### How should this be manually tested?
- Pull this branch `add-about-image`
- Navigate to the `index.html` file
- Scroll to the about section to see the image implementation
#### Screenshots (if appropriate)
<img width="1439" alt="Screen Shot 2019-04-30 at 2 41 44 PM" src="https://user-images.githubusercontent.com/42177767/56965957-22fee380-6b56-11e9-9d60-f9a9229147bb.png">
